### ISSUE TITLE
About Framepack Model Train Speed

### ISSUE THREAD

**Issue by Password1314** [kohya-ss/musubi-tuner#650]
I'm currently training FramePack on a single RTX 4090 (24GB). My training set consists of 640 videos, each with a resolution of 640×640 and 181 frames (one frame per 30 seconds). When running on my GPU, the model is currently running at a speed of 30 seconds per iteration. Is this training speed normal? I'm using the following configuration file:
`CUDA_VISIBLE_DEVICES=1 nohup accelerate launch \
    --num_cpu_threads_per_process 1 \
    --mixed_precision bf16 \
    src/musubi_tuner/fpack_train_network.py \
    --dit /mnt/ssd7t/syl/video_model_weight/DiT_Model/FramePackI2V_HY_bf16.safetensors \
    --vae /mnt/ssd7t/syl/video_model_weight/VAE/diffusion_pytorch_model.safetensors \
    --text_encoder1 /mnt/ssd7t/syl/video_model_weight/LLaMA/model-00001-of-00004.safetensors \
    --text_encoder2 /mnt/ssd7t/syl/video_model_weight/CLIP/model.safetensors \
    --image_encoder /mnt/ssd7t/syl/video_model_weight/SigLIP/model.safetensors \
    --dataset_config /home/syl/Project/musubi-tuner/dataset_config_short.toml \
    --vae_chunk_size 8 \
    --vae_spatial_tile_sample_min_size 64 \
    --flash_attn \
    --mixed_precision bf16 \
    --optimizer_type adamw8bit \
    --learning_rate 2e-4 \
    --gradient_checkpointing \
    --gradient_checkpointing_cpu_offload \
    --timestep_sampling shift \
    --weighting_scheme none \
    --discrete_flow_shift 3.0 \
    --max_data_loader_n_workers 2 \
    --persistent_data_loader_workers \
    --network_module networks.lora_framepack \
    --network_dim 32 \
    --max_train_epochs 16 \
    --save_every_n_epochs 2 \
    --seed 42 \
    --output_dir /mnt/ssd7t/syl/Output/output_short \
    --output_name short-lora \
    --fp8_base \
    --fp8_scaled \
    --fp8_llm > /mnt/ssd7t/syl/Output/output_short/training.log 2>&1 &`

<img width="1662" height="298" alt="Image" src="https://github.com/user-attachments/assets/c3cc92c2-2dc5-46a3-8c09-33020358f63d" />

**Comment by kohya-ss** [comment]
It also depends on the dataset configuration, so could you please share your dataset configuration file?

**Comment by Password1314** [comment]
> It also depends on the dataset configuration, so could you please share your dataset configuration file?

Thank you for your reply. My dataset configuration is as follows：
```
[general]
resolution = [640, 640]
batch_size = 1
enable_bucket = false
caption_extension = ".txt"
bucket_no_upscale = false


[[datasets]]
video_directory = "/mnt/ssd7t/syl/RD_data/train_videos"
cache_directory = "/mnt/ssd7t/syl/RD_data/cache_latents_short"
caption_extension = ".txt"
frame_extraction = "full"
source_fps = 30.0
max_frames = 181
fp_latent_window_size = 9
num_repeats = 1
```
Additionally, I am training the model on an Ubuntu system, and during training, the GPU memory usage is approximately 23GB.

**Comment by kohya-ss** [comment]
I've tested on RTX A6000 with the same dataset settings, the time for one step was about 25 seconds. `--gradient_checkpointing_cpu_offload` is a little slow, so if you remove it, and specify a smaller value for `--blocks_to_swap`, such as 5-10, within the range of your VRAM, the total speed may be faster.

**Comment by Password1314** [comment]
Thank you so much for your answer. Thank you very, very much!​
